src/components/MessageList/MessageListInner.js

```diff
@@ -289,7 +290,7 @@ const MessageListInner = (props) => {
       {...internalInfiniteScrollProps}
     >
       <ul className="str-chat__ul">{elements}</ul>
-
+      {!threadList && <TypingIndicator />}
```

Comment:
would be nice to also cover this with some tests.

Reply:
agree, we don't have any tests set up for the MessageList yet so out of the scope of this PR